src/modules/campaign-monitor/services/campaign-monitor.service.ts

```diff
@@ -84,6 +84,7 @@ export class CampaignMonitorService {
 		});
 
 		if (recipients.length === 0) {
+			this.logger.error('Mail will not be sent - no recipients');
```

Comment:
this is not enough info to debug the issue
also include the whole emailInfo object